lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -1201,14 +1232,21 @@ void LowerClassesPass::lowerClass(om::ClassOp classOp, FModuleLike moduleLike,
     fieldValues.push_back(argumentValue);
   }
 
+  OpBuilder builder = OpBuilder::atBlockEnd(classOp.getBodyBlock());
   classOp.addNewFieldsOp(builder, fieldLocs, fieldValues);
 
   // If the module-like is a Class, it will be completely erased later.
   // Otherwise, erase just the property ports and ops.
   if (!isa<firrtl::ClassLike>(moduleLike.getOperation())) {
     // Erase ops in use before def order, thanks to FIRRTL's SSA regions.
-    for (auto *op : llvm::reverse(opsToErase))
-      op->erase();
+    for (auto &op :
+         llvm::make_early_inc_range(llvm::reverse(moduleLike.getOperation()
+                                                      ->getRegion(0)
+                                                      .getBlocks()
+                                                      .front()
+                                                      .getOperations())))
+      if (isPropertyOp(op) && !isa<InstanceOp>(op))
+        op.erase();
```

Comment:
Just making sure, is there anything to erase in the original module, since we're moving all the property operations?

Reply:
Good point, I think this is dead code now.